src/pages/jotai-onyx/JotaiOnyxScreen.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import {View, Text} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import {ONYX_KEYS} from '../../lib/onyx-keys';
+
+type Props = {
+  betas: string[];
+};
+
+function JotaiOnyxScreen({betas}: Props) {
+  return (
+    <View>
+      {betas ? betas.map(beta => <Text>{beta}</Text>) : <Text>No betas</Text>}
```

Comment:
need to check for `betas.length`, `[]` will fail now

Reply:
just placeholder code